### PR TITLE
Feature/gradle

### PR DESCRIPTION
**Pull request by ravitej-aluru**
- [x] Added new folder for next sdk. Called it symbol-direct-java-sdk

- [x] Optimised gradle build script of symbol direct java sdk

- [x] Created gradle build for integration tests

- [x] Removed next submodule from integration tests

- [x] Created script for travis CI

### CODE REVIEW

**Comment by ravitej-aluru** [comment]
Hey @Wayonb, some clean-up remaining like:

- Removing pom.xml

- Cleaning build.gradle in integration tests and symbol direct java sdk

I will do that once you can confirm it is all good, just to be on the safe side.

**Comment by ravitej-aluru** [comment]
@Wayonb, this PR is ready for your review please.

File: integrationtests/src/test/java/io/nem/symbol/automationHelpers/helper/BaseHelper.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package io.nem.symbol.automationHelpers.helper;
 
-import gherkin.lexer.De;
+// import gherkin.lexer.De;
```
Comment (Wayonb) [inline integrationtests/src/test/java/io/nem/symbol/automationHelpers/helper/BaseHelper.java:3]: remove if not needed

File: symbol-direct-java-sdk/build.gradle [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+    // }
+// }
+
+// plugins {
+    // id 'jvm-component'
+    // id 'java'
+    // id 'net.saliman.properties' version '1.4.6'
+    // id 'jacoco'
+    // id 'com.github.kt3k.coveralls' version '2.8.2'
+// }
```
Comment (Wayonb) [inline symbol-direct-java-sdk/build.gradle:33]: remove if not needed

File: symbol-direct-java-sdk/build.gradle [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+//     description = 'Runs integration tests.'
+//     dependsOn testClasses
+//     shouldRunAfter test
+//     classpath = sourceSets.integrationTest.runtimeClasspath
+
+//     main = 'org.junit.platform.console.ConsoleLauncher'
+//     args = ['--scan-class-path',
+//             sourceSets.integrationTest.output.classesDir.absolutePath,
+//             '--reports-dir', "${buildDir}/test-results/junit-integrationTest"]
+// }
```
Comment (Wayonb) [inline symbol-direct-java-sdk/build.gradle:80]: remove if not needed

File: symbol-direct-java-sdk/projects-status.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
+# Projects & SDK Protocol Compatibility
```
Comment (Wayonb) [inline symbol-direct-java-sdk/projects-status.md:1]: remove this file

File: integrationtests/gradlew [hunk 5]
```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env sh
```
Comment (Wayonb) [inline integrationtests/gradlew:1]: Should all these gradle files be on the root? like gradle/wrapper folder so each project does need their own copy?

Comment (ravitej-aluru) [reply integrationtests/gradlew]: done

**Pull request merged**